packages/dashboard/src/pages/Wallets/components/Transfer/TransferERC20.tsx

```diff
@@ -59,9 +67,26 @@ export const TransferERC20 = memo<TransferERC20Props>(({ token }) => {
     const nativeTokenPrice = useNativeTokenPrice()
     const isNativeToken = selectedToken.type === EthereumTokenType.Native
 
+    //#region resolve ENS domain
+    const {
+        value: registeredAddress = '',
+        error: resolveEnsDomainError,
+        loading: resolveEnsLoading,
```

Comment:
It could be other naming services.

```suggestion
        error: lookupAddressError,
        loading: lookupAddressLoading,
```